File: Coding Exercises/TypeScript/Day56/areEqual.ts
function areEqual(arg1: any, arg2: any): boolean {
  if (arg1 === null && arg2 === null) {
      return true;
  }

  if ((arg1 === null && arg2 !== null) || (arg1 !== null && arg2 === null)) {
      return false;
  }

  return arg1 === arg2;
}

File: Coding Exercises/Python/Day56/are_equal.py
def are_equal(arg1, arg2):
    if arg1 is None and arg2 is None:
        return True

    if (arg1 is None and arg2 is not None) or \
            (arg1 is not None and arg2 is None):
        return False

    return arg1 == arg2
